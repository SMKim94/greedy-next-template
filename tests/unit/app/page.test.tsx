import { render, screen } from '@testing-library/react';
import Page from '@/app/page';

describe('메인 페이지', () => {
  test('메인 페이지에 제목이 표시되어야 합니다', () => {
    render(<Page />);
    const title = screen.getByText('Hello, World!');
    expect(title).toBeInTheDocument();
  });
});

describe('Page 컴포넌트', () => {
  it('Page 컴포넌트가 스냅샷과 일치해야 합니다', () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
