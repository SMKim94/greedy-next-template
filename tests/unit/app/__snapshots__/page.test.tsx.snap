// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page 컴포넌트 Page 컴포넌트가 스냅샷과 일치해야 합니다 1`] = `
<div>
  <h1
    class="text-2xl font-thin underline"
  >
    Hello, World!
  </h1>
</div>
`;
